libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,7 @@ namespace // anonymous
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 6 * 1024;
+static size_t const c_singleExecutionStackSize = 13 * 1024;
```

Comment:
Unfortunately I don't know where it comes from, but it is not the case for Debug builds. I can git bisect, but I don't know any tool that could analyse stack space usage.